cfgov/jinja2/v1/index.html

```diff
@@ -215,9 +272,11 @@ <h3 class="h4">
                         </section>
                     {% endfor %}
                 </div>
+
+                {% endif %}
+
             </div>
         </div>
-
```

Comment:
Trivial: Consider also removing the blank line after the opening `<main>` tag.

Reply:
Removed in b900f64